### PR TITLE
Etesync dav 0.20.0

### PR DESCRIPTION
**Pull request by ProjectMoon**
Continuation of https://github.com/void-linux/void-packages/pull/22810.

### CODE REVIEW

**Comment by ProjectMoon** [comment]
Still working on implementing suggested changes from original PR.

**Comment by lemmi** [comment]
So after thinking about this a little bit, we should follow @CameronNemo suggestion and only have a single `radicale` package. It's what we usually do and only in very specific cases do we manage multiple versions this way. Thanks for adopting the package.

File: srcpkgs/radicale/template [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
-maintainer="lemmi <lemmi@nerd2nerd.org>"
+maintainer="projectmoon <projectmoon@agnos.is>"
 license="GPL-3.0-or-later"
 homepage="https://radicale.org"
-distfiles="${PYPI_SITE}/R/Radicale/Radicale-${version}.tar.gz"
-checksum=c007198ea45ef797344672c681d4c13f8b4aa85c15c41a1156225767a405c92b
-
+distfiles="https://github.com/Kozea/Radicale/archive/${version}.tar.gz"
+checksum=9e22273dda13938b44e0555a96f63e0491e42c1e58a290864ecae1181580be8c
+provides="radicale-${version}_${revision}"
```
Comment (ahesford) [inline srcpkgs/radicale/template:18]: This line doesn't add value.

**Review by ahesford — CHANGES_REQUESTED** [review]
#24945 comes closer to doing the right thing with `radicale`. Specifically, it seems there are some new access control files that should be installed, and you've dropped the service directory. Either merge the changes from that PR into this one (also addressing my issues there) or drop the `radicale` changes and wait for #24945 to be merged, leaving this PR to update only `etesync-dav`.

**Comment by ProjectMoon** [comment]
Waiting for the other PR is definitely the best path for me. I don't really have time to deal with upgrading radicale at the same time.

**Comment by ahesford** [comment]
`radicale` is in, so you can strip this down to the `etesync-dav` update only.

**Pull request closed without merging**